airflow/www/auth.py

```diff
@@ -44,6 +46,27 @@ def get_access_denied_message():
     return conf.get("webserver", "access_denied_message")
 
 
+def has_access(permissions: Sequence[tuple[str, str]] | None = None) -> Callable[[T], T]:
+    """
+    Factory for decorator that checks current user's permissions against required permissions.
+
+    Deprecated. Do not use this decorator, use one of the decorator `has_access_cluster_*` defined in
+    airflow/www/auth.py instead.
+
+    This decorator is widely used in user plugins, do not remove it. See
+    https://github.com/apache/airflow/pull/33213#discussion_r1346287224
+    """
+    warnings.warn(
+        "The 'has_access' decorator is deprecated. Please use one of the decorator `has_access_cluster_*`"
```

Comment:
Thanks for pointing to the new API points, I assume there is a small type of the prefix used?
```suggestion
    Deprecated. Do not use this decorator, use one of the decorator `has_access_*` defined in
    airflow/www/auth.py instead.

    This decorator is widely used in user plugins, do not remove it. See
    https://github.com/apache/airflow/pull/33213#discussion_r1346287224
    """
    warnings.warn(
        "The 'has_access' decorator is deprecated. Please use one of the decorator `has_access_*`"
```

Reply:
Oh yeah, you're right. Nice catch!